src/GitHub.App/Services/RepositoryCloneService.cs

```diff
@@ -121,20 +172,7 @@ public async Task<ViewerRepositoriesModel> ReadViewerRepositories(HostAddress ad
             try
             {
                 await vsGitServices.Clone(cloneUrl, repositoryPath, true, progress);
-
                 await usageTracker.IncrementCounter(x => x.NumberOfClones);
```

Comment:
This method is being called by `CreateRepository` here:
https://github.com/github/VisualStudio/blob/8067273b31420d5f9d487fcd29d35421d50e68c8/src/GitHub.App/Services/RepositoryCreationService.cs#L40